### DISCUSSION TITLE
Flet Studio vs Android Studio

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [flet-dev/flet#993] [General]
Will Flet Studio be like Android Studio

**Comment by iron3oxide** [comment]
Your question is a bit unspecific, but I'd say no. It will not be an IDE, but rather a testing environment. See [this blog post section](https://flet.dev/blog/flet-mobile-strategy/#flet-studio-for-ios-and-android)

**Reply by saurabhwadekar** [reply]
Thanks ❤️❤️❤️
